### ISSUE TITLE
Custom bash completions doesn't work

### ISSUE THREAD

**Issue by xakrume** [warpdotdev/Warp#2639]
### Describe the solution you'd like?

```text
Im using my own completions for SSH:


# SSH aliases
_ssh() 
{
    local cur prev opts
    COMPREPLY=()
    cur="${COMP_WORDS[COMP_CWORD]}"
    prev="${COMP_WORDS[COMP_CWORD-1]}"
    opts=$(grep '^Host' ~/.ssh/config | grep -v '[?*]' | cut -d ' ' -f 2-)

    COMPREPLY=( $(compgen -W "$opts" -- ${cur}) )
    return 0
}
complete -F _ssh ssh
```

Completions from .bash_profile doesn't work in Warp.
```


### Is your feature request related to a problem? Please describe.

_No response_

### Additional context

_No response_

### How important is this feature to you?

5 (Can't work without it!)

### Warp Internal (ignore) - linear-label:770f6576-d6c0-4e4f-a259-fc64b5156087

None

**Comment by dannyneira** [comment]
Thanks for submitting @xakrume This is something we're working to improve and you can track more progress here: #652

**Comment by dannyneira** [comment]
TLDR is we're looking to add the ability for custom completion spec for our users to add onto Warp's existing completions. In the meantime here's the full list of [completions we support](https://docs.warp.dev/features/completions).

Please 👍 and subscribe to that request as it helps us gauge interest. We'll post any updates there.

Closing as duplicate.
